repo/README.md

```diff
@@ -32,46 +31,35 @@ Repo Implementations:
 
```

Comment:
It would be pretty sweet to upgrade https://github.com/maxogden/abstract-blob-store to have a parallel interface for Go, so that we can have all of those s3/fs/mem repos for 'free'.